src/schema/fairs.ts

```diff
@@ -60,7 +60,8 @@ const Fairs: GraphQLFieldConfig<void, ResolverContext> = {
       gravityOptions.id = options.ids
       delete gravityOptions.ids
     }
-    return fairsLoader(gravityOptions)
+    const { body: fairs } = await fairsLoader(gravityOptions)
+    return fairs
```

Comment:
Hmm, was this simply returning a Promise before this change 🤔 ?

Reply:
Correct